packages/api/cms-api/src/page-tree/page-tree-node-document-entity-scope.service.ts

```diff
@@ -0,0 +1,23 @@
+import { Injectable } from "@nestjs/common";
+
+import { DocumentInterface } from "../document/dto/document-interface";
+import { EntityScopeServiceInterface } from "../user-permissions/decorators/scoped-entity.decorator";
+import { ContentScope } from "../user-permissions/interfaces/content-scope.interface";
+import { PageTreeService } from "./page-tree.service";
+
+@Injectable()
+export class PageTreeNodeDocumentEntityScopeService implements EntityScopeServiceInterface<DocumentInterface> {
+    constructor(private readonly pageTreeService: PageTreeService) {}
```

Comment:
better would be to inject `PageTreeReadApiService`, but as we know from EntityInfo this is not so easy to do correctly, because that service is Request scoped